Parallels/ParallelsDesktop.download.recipe

```diff
@@ -38,7 +38,7 @@ This recipe differs from the one in keeleysam-recipes because it offers code sig
 			<key>Arguments</key>
 			<dict>
 				<key>input_path</key>
-				<string>%pathname%/Parallels Desktop.app</string>
+				<string>%pathname%/%NAME%.app</string>
 				<key>requirement</key>
```

Comment:
I doubt the installed application name will ever be different from "Parallels Desktop.app". If you change NAME to "Unlicensed-Parallels Desktop" the CodeSignatureVerification step will fail since it will be looking for the wrong app name...